.github/setup-unit.sh

```diff
@@ -63,4 +63,7 @@ kestra:
 EOF
 
 echo "Setup complete - PAT: $GITEA_PAT"
-echo "Repository URL: https://localhost:3443/gitea_admin/kestra-test.git"
\ No newline at end of file
+echo "Repository URL: https://localhost:3443/gitea_admin/kestra-test.git"
+
+# CLEANUP
+rm -rf certs
```

Comment:
This won't work since it has to be used afterwards by the tests.
I already added a [clean-up step](https://github.com/kestra-io/actions/pull/6/files) in the `actions` repository for that.